copy.go

```diff
@@ -86,18 +86,12 @@ func copyNextOrSkip(src, dest string, info os.FileInfo, opt Options) error {
 // with considering existence of parent directory
 // and file permission.
 func fcopy(src, dest string, info os.FileInfo, opt Options) (err error) {
-
-	var readcloser io.ReadCloser
-	if opt.FS != nil {
-		readcloser, err = opt.FS.Open(src)
-	} else {
-		readcloser, err = os.Open(src)
-	}
+	readcloser, err := fopen(src, opt)
 	if err != nil {
-		if os.IsNotExist(err) {
-			return nil
+		if !os.IsNotExist(err) {
+			return
```

Comment:
[Just IMO for readability] 🤔 
In this case, `if os.IsNotExist(err)` is the exceptional case.
Code-readers expect `if err != nil, then return err` as usual. But the interesting point is that we do not return the error if it's `ErrNotExist`.
To represent this flow of thinking, I do prefer using `if os.IsNotExist(err); then return nil`.

In addition, `!os.IsNotExist(err)` is a double-negative way.